LambdaSource/Scotty_TableSlotUpdater/lambdaHandler.py

```diff
@@ -1,10 +1,98 @@
 #! python3
 import boto3
-import argparse
-
+import time
 from pprint import pprint
 
 
+def updateBot(lex, updatedSlotVersion):
```

Comment:
Given that this method is used to update intents that have a 'table' slot in the 'Scotty' bot, and that we talked about renaming this function to be a more generic 'SlotUpdater', perhaps you could rename this method?